sphinx/ext/doctest.py

```diff
@@ -38,7 +38,7 @@
 logger = logging.getLogger(__name__)
 
 blankline_re = re.compile(r'^\s*<BLANKLINE>', re.MULTILINE)
-doctestopt_re = re.compile(r'#\s*doctest:.+$', re.MULTILINE)
+doctestopt_re = re.compile(r'\s*#\s*doctest:.+$', re.MULTILINE)
```

Comment:
Do we include new lines here? If not, perhaps:

```suggestion
doctestopt_re = re.compile(r'[ \t]*#\s*doctest:.+$', re.MULTILINE)
```